roles/contiv/tasks/pkgMgrInstallers/centos-install.yml

```diff
@@ -16,15 +16,15 @@
   tags:
     - ovs_install
 
-- name: PkgMgr CentOS | Install openstack kilo rpm
-  yum: name=/tmp/rdo-release-kilo-2.noarch.rpm state=present
+- name: PkgMgr RHEL/CentOS | Install openstack ocata rpm
+  yum: name=/tmp/rdo-release-ocata-2.noarch.rpm state=present
   tags:
     - ovs_install
 
-- name: PkgMgr CentOS | Install ovs
+- name: PkgMgr RHEL/CentOS | Install ovs
   yum:
-    pkg=openvswitch
-    state=latest
+    pkg=openvswitch-2.5.0-2.el7.x86_64
```

Comment:
rhel repo don't have this ovs version. Is it ok to use these version?
yum list --showduplicates openvswitch
Loaded plugins: langpacks, product-id, search-disabled-repos, subscription-manager
Available Packages
openvswitch.x86_64                              1:2.5.0-5.git20160628.el7fdb                               rhel-7-fast-datapath-htb-rpms
openvswitch.x86_64                              1:2.5.0-14.git20160727.el7fdb                              rhel-7-fast-datapath-htb-rpms